components/CreateForm/PreviewModal.tsx

```
@@ -0,0 +1,124 @@
+            // eslint-disable-next-line
+            <Image
+              layout="fill"
+              objectFit="cover"
+              objectPosition="center"
+              src={getFetchableUrl(previewUri) || ""}
+              alt="not found preview."
+            />
+          ) : (
+            <div className="size-full p-3 flex justify-center items-center">
+              <p className="font-spectral text-3xl">
+                {Boolean(progress) ? `${progress}%` : "No Preview."}
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)